### PR TITLE
update mentee and mentor links and copy on main page button

### PR DESCRIPTION
**Pull request by ScizorsBLBC**
## 🚀 What's New
Added new links for mentee and mentor and new copy on main page button to learn more about the programs

Story in Jira:
https://emergentworks.atlassian.net/jira/software/c/projects/CORE/boards/2?modal=detail&selectedIssue=WEB-139

## 📸 Screenshots (if applicable)

image

## 📝 Additional information

### CODE REVIEW

File: src/pages/index.js [hunk 1]
```diff
@@ -182,7 +185,8 @@ const IndexPage = () => {
               </Link>
             </li>
           </ul>
-        </section>x
+        </section>
+        x
```
Comment (carosee) [inline src/pages/index.js:189]: this isnt related to anything you changed, but I think someone accidentally left an extraneous character here and you can see it on the website- can we remove the `x`?

Screen Shot 2023-02-21 at 3 12 07 PM

**Review by carosee — APPROVED** [review]
left a request for a minor unrelated change, but this looks great otherwise! thanks!

File: src/pages/index.js (hunk 1 above)
Comment (ScizorsBLBC) [reply src/pages/index.js:189]: oh weird that x bottom left outside the box lol

Comment (ScizorsBLBC) [reply src/pages/index.js]: Ok @carosee that should be ready to review now

**Review by carosee — APPROVED** [review]

**Pull request merged**